docs/content/en/docs/pipeline-stages/builders/buildpacks.md

```diff
@@ -1,39 +1,66 @@
 ---
-title: "CNCF Buildpacks"
+title: "Cloud Native Buildpacks"
 linkTitle: "Buildpacks"
 weight: 50
 featureId: build.buildpacks
 ---
 
-[Buildpacks]((https://buildpacks.io/)) enable building language-based containers from source code, without the need for a Dockerfile.
+[Cloud Native Buildpacks](https://buildpacks.io/) enable building language-based
+containers from source code, without the need for a Dockerfile.
 
-Skaffold supports building with buildpacks natively.
- 
-Skaffold buildpacks support, builds the image inside local docker daemon. 
-It mounts the source dependencies and local artifact cache if caching is enabled 
-to a container in a docker daemon. These get unmounted once the build process is finished.
+Skaffold supports building with Buildpacks natively, without installing
+additional software. However, Buildpacks require a local Docker daemon
+to be used.
```

Comment:
Docker is additional software :-)

_Skafold supports building with Cloud Native Buildpacks, a source to image solution that requires only a local Docker installation._

?